initScripts/x86_64/Ubuntu_14.04/Docker_18.03.sh

```diff
@@ -0,0 +1,395 @@
+#!/bin/bash
+set -e
+set -o pipefail
+
+# initScript for Ubuntu 16.04 and Docker 18.03
+# ------------------------------------------------------------------------------
+
+readonly DOCKER_VERSION="18.03.0"
```

Comment:
we should probably install [`18.03.1`](https://docs.docker.com/release-notes/docker-ce/) as it has some bug fixes